app/src/main/java/com/devshish/internship/presentation/service/player/MediaSessionCallback.kt

```diff
@@ -1,21 +1,31 @@
 package com.devshish.internship.presentation.service.player
 
+import android.content.Context
 import android.net.Uri
 import android.os.Bundle
 import android.support.v4.media.session.MediaSessionCompat
 import android.support.v4.media.session.PlaybackStateCompat
+import androidx.core.app.NotificationCompat
+import androidx.core.content.ContextCompat
+import androidx.media.session.MediaButtonReceiver
+import com.devshish.internship.R
 import com.devshish.internship.presentation.ui.utils.song
 import com.devshish.internship.presentation.ui.utils.toMediaMetadata
 import com.google.android.exoplayer2.ExoPlayer
 import com.google.android.exoplayer2.MediaItem
 import timber.log.Timber
 
 class MediaSessionCallback(
+    private val context: Context,
```

Comment:
I think that MediaSessionCallback is becoming rather huge and unreadable - so it would be good to extract notification logic in some class , and provide this class here (instead of Context)